app/views/static/home.html.erb

```diff
@@ -1,11 +1,187 @@
-Erudition
-</br>
-</br>
-<% if !user_signed_in? %>
-  <%= link_to 'Sign up', new_user_registration_path %>
-  </br>
-  <%= link_to 'Log in', new_user_session_path %>
-<% end %>
-<% if user_signed_in? %>
-  <%= link_to 'Log out', destroy_user_session_path, method: :delete %>
-<% end %>
+<div class="page page-home">
+
+  <nav class="navbar navbar-default">
+    <div class="container">
+      <div class="navbar-header">
+        <a class="navbar-brand" href="/">Erudition</a>
+      </div>
+      <div class="navbar-header navbar-right">
+        <% if !user_signed_in? %>
+          <%= link_to 'Log in', new_user_session_path, class: 'btn btn-default navbar-btn nav-pills' %>
+        <% end %>
+        <% if user_signed_in? %>
+          <p class="navbar-text">Signed in as email@email.com</p>
+          <%= link_to 'Log out', destroy_user_session_path, method: :delete, class: 'btn btn-default navbar-btn' %>
+        <% end %>
+      </div>
+    </div>
+  </nav>
```

Comment:
Totally agree, however having this be a static splash page I would like to get some feedback on how the team wants to use the page before defining any partials.

Is it going to be a one off page hooked up strictly for A/B testing or are we going to be able to re-use elements like the ```nav``` ```footer``` etc throughout the app?